tests/flytekit/unit/core/test_serialization.py

```diff
@@ -368,13 +370,51 @@ def parent_wf() -> (int, int, int, int):
 
 def test_serialization_named_outputs_single():
     @task
-    def t1() -> str:
+    def t1() -> typing.NamedTuple("OP", a=str):
         return "Hello"
```

Comment:
so here we're just wrapping it for them?  not raising an error not forcing them to return `("Hello",)`?

Reply:
Yup